apps/api/src/server/app/state-machine/household-appeal.machine.test.js

```diff
@@ -16,42 +19,54 @@ test.before('sets up mocking of actions', () => {
-	['overdue_lpa_questionnaire', 'RECEIVED', 'received_lpa_questionnaire', true],
-	['overdue_lpa_questionnaire', 'COMPLETE', 'overdue_lpa_questionnaire', false],
-	['overdue_lpa_questionnaire', 'INCOMPLETE', 'overdue_lpa_questionnaire', false],
-	['received_lpa_questionnaire', 'COMPLETE', 'available_for_investigator_pickup', true],
-	['received_lpa_questionnaire', 'INCOMPLETE', 'incomplete_lpa_questionnaire', true],
-	['incomplete_lpa_questionnaire', 'COMPLETE', 'available_for_investigator_pickup', true],
-	['available_for_investigator_pickup', 'PICKUP', 'site_visit_not_yet_booked', true]
+	['received_appeal', 'INVALID', 'invalid_appeal', true, { appealId: 1 }],
+	['received_appeal', 'VALID', 'awaiting_lpa_questionnaire', true, { appealId: 1 }],
+	['received_appeal', 'INFO_MISSING', 'awaiting_validation_info', true, { appealId: 1 }],
+	['awaiting_validation_info', 'INVALID', 'invalid_appeal', true, { appealId: 1 }],
+	['awaiting_validation_info', 'INFO_MISSING', 'awaiting_validation_info', false, { appealId: 1 }],
+	['awaiting_validation_info', 'VALID', 'awaiting_lpa_questionnaire', true, { appealId: 1 }],
+	['invalid_appeal', 'INVALID', 'invalid_appeal', undefined, { appealId: 1 }],
+	['invalid_appeal', 'INFO_MISSING', 'invalid_appeal', undefined, { appealId: 1 }],
+	['invalid_appeal', 'VALID', 'invalid_appeal', undefined, { appealId: 1 }],
+	['awaiting_lpa_questionnaire', 'INVALID', 'awaiting_lpa_questionnaire', false, { appealId: 1 }],
+	['awaiting_lpa_questionnaire', 'INFO_MISSING', 'awaiting_lpa_questionnaire', false, { appealId: 1 }],
+	['awaiting_lpa_questionnaire', 'VALID', 'awaiting_lpa_questionnaire', false, { appealId: 1 }],
+	['awaiting_lpa_questionnaire', 'OVERDUE', 'overdue_lpa_questionnaire', true, { appealId: 1 }],
+	['awaiting_lpa_questionnaire', 'RECEIVED', 'received_lpa_questionnaire', true, { appealId: 1 }],
+	['overdue_lpa_questionnaire', 'RECEIVED', 'received_lpa_questionnaire', true, { appealId: 1 }],
+	['overdue_lpa_questionnaire', 'COMPLETE', 'overdue_lpa_questionnaire', false, { appealId: 1 }],
+	['overdue_lpa_questionnaire', 'INCOMPLETE', 'overdue_lpa_questionnaire', false, { appealId: 1 }],
+	['received_lpa_questionnaire', 'COMPLETE', 'available_for_investigator_pickup', true, { appealId: 1 }],
+	['received_lpa_questionnaire', 'INCOMPLETE', 'incomplete_lpa_questionnaire', true, { appealId: 1 }],
+	['incomplete_lpa_questionnaire', 'COMPLETE', 'available_for_investigator_pickup', true, { appealId: 1 }],
+	['available_for_investigator_pickup', 'PICKUP', 'site_visit_not_yet_booked', true, { appealId: 1 }],
+	['site_visit_not_yet_booked', 'BOOK', 'site_visit_booked', true, { appealId: 1, inspectionType: 'accompanied' }],
+	['site_visit_not_yet_booked', 'BOOK', 'site_visit_booked', true, { appealId: 1, inspectionType: 'access required' }],
+	['site_visit_not_yet_booked', 'BOOK', 'site_visit_booked', true, { appealId: 1, inspectionType: 'any other type' }]
```

Comment:
Should this inspectionType be 'unaccompanied' (as per the description in the prototype)?

Reply:
I can certainly add that in the tests